src/main/java/storage/Storage.java

```diff
@@ -11,9 +11,12 @@
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
+import java.util.logging.Level;
+import java.util.logging.Logger;
 
 public class Storage {
 
+    private static Logger logger = Logger.getLogger("Storage");
     public static final Path FILE_PATH = Path.of("./save/tasks.txt");
```

Comment:
Looks like this line is causing a checkstyle error. Try referring to this post: https://stackoverflow.com/questions/6391540/what-does-variable-access-definition-in-wrong-order-mean-in-checkstyle to see the correct order of declaration.